modules/react/breadcrumbs/lib/Breadcrumbs.tsx

```diff
@@ -82,14 +82,15 @@ export const Breadcrumbs = createContainer('nav')({
      * [View Developer Docs](https://canvas.workdaydesign.com/components/navigation/breadcrumbs/#breadcrumbsoverflowbutton)
      *
      * The toggle button for the Breadcrumbs Menu.
-     * This button is rendered implicitly in the `Breadcrumbs.List` when the list overflows.
-     * However, if you need to pass props to it, you can do so by passing `overflowButtonProps` to the List.
+     * This button is rendered in the `Breadcrumbs.List` when `overflowButtonProps` passed
+     * and becomes visible when the list overflows. `overflowButtonProps must contain at least `aria-label`.
+     * If you need to pass other props to it, you can do so by adding it to `overflowButtonProps` passed to the List.
      *
      * @example
      * ```tsx
      * <Breadcrumbs.List
      *   overflowButtonProps={{
-     *     'aria-label': 'More breadcrumb links',
+     *     'aria-label': 'More links',
      *     onClick={handleOverflowButtonClick}
      *   }}
```

Comment:
Question about how the overflow button works in Breadcrumbs:

The overflow button in Tabs accepts a component while Breadcrumbs accepts props:

```jsx
// Tabs
<Tabs.List overflowButton={<Tabs.OverflowButton>More</Tabs.OverflowButton>} />

// Breadcrumbs
<Breadcrumbs.List overflowButtonProps={{...}} />
```

The Tabs approach allows more configuration via JSX. The Breadcrumbs approach is easier, but less powerful.

Reply:
Agree with this. I've updated `Breadcrumbs.List` to keep this consistent.